### ISSUE TITLE
rpm package creation

### ISSUE THREAD

**Issue by ccacciari** [EUDAT-B2SAFE/B2SAFE-core#85]
In order to build the rpm package I was forced to remove manually one of the two alternative dependencies from the following line in the file packaging/irods-eudat-b2safe.spec:

`Requires:       irods-icat or irods-serer`

otherwise I get (with irods 4.1.8):

```
$ rpm -Uv /home/irodsmaster/rpmbuild/RPMS/noarch/irods-eudat-b2safe-4.0-0.noarch.rpm
error: Failed dependencies:
    irods-server is needed by irods-eudat-b2safe-4.0-0.noarch
    or is needed by irods-eudat-b2safe-4.0-0.noarch
```

So the rpm package manager does not interpret "or" as a logical operator, but as a package.
I'm using:
```
$ rpm -v
RPM version 4.11.3
Copyright (C) 1998-2002 - Red Hat, Inc.
```

**Comment by cookie33** [comment]
I also made an error. It says irods-serer and NOT irods-server

**Comment by cookie33** [comment]
The or functionality is in there since rpm version 4.13. On Centos 6 it is still 4.8.

So we should look if a file is present: `/etc/irods/service_account.config`. This means iRODS is installed AND configured.

**Comment by cookie33** [comment]
On Centos you can upgrade using `rpm -Uvh --nodeps` to ignore the dependancies.

**Comment by cookie33** [comment]
See #88. remove required for now from spec file.

**Comment by cookie33** [comment]
On Debian it works as advertised. 👍

**Comment by cookie33** [comment]
Fixed in level branch

**Comment by cookie33** [comment]
fixed in v4.0-1
